overlord/ifacestate/handlers.go

```diff
@@ -349,6 +346,47 @@ func (m *InterfaceManager) undoDiscardConns(task *state.Task, _ *tomb.Tomb) erro
 	return nil
 }
 
+func getDynamicHookAttributes(task *state.Task) (map[string]interface{}, map[string]interface{}, error) {
```

Comment:
This needs result names. As it is we need to read the code to find out what the results mean.

Doing that also allows avoiding the type repetition ( as in _(a, b type)_)